docs/examples/color_channels/rgb_grayscale.jl

```diff
@@ -0,0 +1,22 @@
+# ---
+# title: RGB to GrayScale
+# cover: assets/rgb_grayscale.png
+# ---
+
+# This example illustrates RGB to Grayscale Conversion
+
+
+using Images, TestImages
+
+rgb_image = testimage("lighthouse")
+
+# I = Gray.(RGB) converts an RGB image to Grayscale. 
+
+gray_image = Gray.(rgb_image)
+hcat(rgb_image, gray_image)
```

Comment:
```diff
- hcat(rgb_image, gray_image)
+ mosaicview(rgb_image, gray_image; nrow = 1)
```

I hope we can advertise the new util `mosaicview` here, which is reexported by `ImageCore`.

https://github.com/JuliaArrays/MosaicViews.jl#compare-two-images